components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -647,22 +666,34 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
         }
     }
 
+    private int handleLimitEqualsNULL(Integer limit){
+
+        // Limit equal to null implies return all users. Return all users scenario handled by the following methods by
+        // expecting count as zero.
+        if (limit == null) {
+            limit = 0;
+        }
+        return limit;
+
```

Comment:
shall we remove unnecessary lines

Reply:
Fixed: df77256698c74c829147f73d00e2f63979199c16